Elves Versus Unused Imports

oalders opened the pull request:
I've already assigned a publish date to this article as we have room in the schedule for it.

oalders commented:
@openstrike, please only review if it's convenient. 😄

openstrike commented on 2023/articles/2023-12-23.pod:
```diff
@@ -0,0 +1,295 @@
+
+If you're interested in an in-depth discussion of how C<perlimports> and Perl's
+C<use> and C<require> work, check out L<perlimports or "Where did that symbol come
+from?"|https://www.youtube.com/watch?v=fKqxdTbGxYY>.
```
It's very much a personal preference but I was reading the rendered POD and followed this link to test it for 404 etc. and was surprised it was a youtube video instead of an article/paper/blog post. It might be kind to those on low bandwidth or metered connections (or grumpy folks like me) just to indicate it is an auto-playing video here. eg. "check out the [link] video"?

openstrike commented on 2023/articles/2023-12-23.pod:
```diff
@@ -0,0 +1,295 @@
+    $ yath t
+    $ perlimports --lint t
+
+And indeed, all of the steps "just worked". We ran the test(s) via L<yath> and
```
Any particular reason why you used yath here rather than `make test`? Might be nice to tell the readers why, even if it's just "brevity". Sometimes I find these little incidental points in the articles turn out to be very useful/interesting. :-)

openstrike reviewed (approved):
Very good. The article is fine as it is - just a couple of thoughts about what I might do but feel free to ignore those especially if time is short.

oalders commented:
@openstrike that was helpful. Thanks! If you have a minute, I've addressed the points and also added a bit about `precious`. I had been planning a blog post entirely on `precious`, but it looks like we should be getting a post in today via #365, so I've just mentioned it "briefly" in this article rather than skipping it entirely.

openstrike reviewed (approved):
Looks great. Interesting to hear about precious - I will have to give that a trial.

oalders commented:
Thanks, @openstrike!
